tvm/python/tvm/autotvm/task/space.py

```diff
@@ -822,6 +826,33 @@ def valid(self):
         """
         return not bool(self.errors)
 
+    def multi_filter(self, **kwargs):
```

Comment:
Which models have multiple filters?

Reply:
yes, we decrease search space by adding multifilter for all axis of local work group in mul must not exceed 1024.